### PR TITLE
signup style #17

### LINKED ISSUE #17: sign up style
#2
**Styling the sign up page**

### PR DESCRIPTION
**Pull request by walaamedhat**
- fixes #17 

- related to #17 

- make signup page html,css

### CODE REVIEW

**Comment by muhmushtaha** [comment]
Let us discuss this tomorrow  @walaamedhat

File: public/signup.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    </form></div>
+  <footer class="footer">2018 Elegant.com © All Rights Reserved</footer>
+</div>
+  </body>
+  <script type="text/javascript">
+  function ShowHideDiv(chkOwner) {
+  var divchkOwner = document.getElementById("divchkOwner");
+  var footer = document.querySelector("footer");
+  divchkOwner.style.display = chkOwner.checked ? "block" : "none";
+  // footer.style.position = chkOwner.checked ? "relative" : "absolute";
```
Comment (muhmushtaha) [inline public/signup.html]: remove the commented line.
use this library atom-beautify to fix indentation 
@walaamedhat

File: public/css/signup.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+.a {
+  text-decoration: none;
+  color: #cf0606;
+}
+.h4 {
+  margin-top: 25px;
+  text-align: center;
+}
+.footer {
+  /*position: absolute;
```
Comment (muhmushtaha) [inline public/css/signup.css]: remove the commented lines

**Review by muhmushtaha — CHANGES_REQUESTED** [review]

File: public/css/signup.css (hunk 2 above)
Comment (walaamedhat) [reply public/css/signup.css]: done

File: public/signup.html (hunk 1 above)
Comment (walaamedhat) [reply public/signup.html]: done

**Review by muhmushtaha — APPROVED** [review]

**Pull request merged**